dynamiqs/integrators/apis/dssesolve.py

```diff
@@ -28,18 +28,21 @@ def dssesolve(
         This function has not been implemented yet. The following API is indicative
         of the planned implementation.
 
+    In quantum optics, the diffusive SSE describes the evolution of a quantum
+    system under homodyne or heterodyne detection with unit detection efficiency.
     This function computes the evolution of the state vector $\ket{\psi(t)}$ at time $t$,
     starting from an initial state $\ket{\psi_0}$ according to the diffusive SSE in Itô
-    form (with $\hbar=1$ and where time is implicit(1))
+    form ($\hbar=1$, time is implicit(1))
     $$
         \begin{split}
-            \dd\ket\psi = \Bigg[
+            \dd\\!\ket\psi = \Bigg[
                 &-iH \dt
                 -\frac12 \sum_{k=1}^N \left(
-                    L_k^\dag L_k - \braket{\psi| L_k + L_k^\dag |\psi} L_k + \braket{\psi| L_k + L_k^\dag |\psi}^2 I
+                    L_k^\dag L_k - \braket{L_k + L_k^\dag} L_k
+                    + \braket{L_k + L_k^\dag }^2 \\! / 4
                 \right) \dt \\\\
                 &+ \sum_{k=1}^N \left(
-                    L_k - \frac12 \braket{\psi| L_k + L_k^\dag |\psi}
+                    L_k - \braket{L_k + L_k^\dag} \\! / 2
```

Comment:
I find it weird to inline some fractions and not some other. I suggest not inlining any fraction.

Reply:
Ok, this one is a personal preference (I think it looks better), but I'm ok to remove.